src/props.gradients.css

```
@@ -1,62 +1,63 @@
 :where(html) {
-  --gradient-1: linear-gradient(to bottom right,#1f005c, #5b0060, #870160, #ac255e, #ca485c, #e16b5c, #f39060, #ffb56b);
-  --gradient-2: linear-gradient(to bottom right,#48005c, #8300e2, #a269ff);
+  --space: ;
```

```suggestion
  --gradient-space: ;
```

sorry for asking you to rename a bunch of stuff… but space seemed super generic, and if we prefix it with --gradient- then it's much clearer what part it plays.

---

Totally agree, especially with HDR colors, it's important to be specific.